apps/frontpage/components/docs/renderers/button.tsx

```diff
@@ -0,0 +1,36 @@
+'use client';
+
+import { cn } from '@utils';
+import { ForwardedRef, forwardRef } from 'react';
+import { ChevronSmallDownIcon } from '@storybook/icons';
+
+interface ButtonProps {
+  onClick?: () => void;
+  title: string;
```

Comment:
I strongly prefer using `children` for this.

Reply:
Done.